easybuild/easyconfigs/s/SciPy-bundle/SciPy-bundle-2019.03-foss-2019a.eb

```diff
@@ -0,0 +1,42 @@
+easyblock = 'PythonBundle'
+
+name = 'SciPy-bundle'
+version = '2019.03'
+
+homepage = 'http://python.org/'
+description = "Bundle of Python packages for scientific software"
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'pic': True, 'lowopt': True}
+
+multi_deps = {'Python': ['3.7.2', '2.7.15']}
+
+use_pip = True
+
+# order is important!
+# package versions updated Mar 15th 2019
+exts_list = [
+    ('numpy', '1.16.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/n/numpy/'],
+        'source_tmpl': '%(name)s-%(version)s.zip',
+        'checksums': ['6c692e3879dde0b67a9dc78f9bfb6f61c666b4562fd8619632d7043fb5b691b0'],
+    }),
+    ('scipy', '1.2.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/s/scipy/'],
+        'checksums': ['e085d1babcb419bbe58e2e805ac61924dac4ca45a07c9fa081144739e500aa3c'],
+    }),
+    ('mpi4py', '3.0.1', {
```

Comment:
One could argue that `mpi4py` shouldn't be part of a `SciPy-bundle`. I don't know if the discussion already happened and it is settled, but I would vote against it. After all, this is the only package that forces this bundle to be installed in a toolchain with MPI.

Reply:
That's not true, `numpy` needs `FFTW` or `imkl`, both of which require MPI (unless you build `impi` crippled without the FFTW wrappers).

Reply:
Well, yes, `numpy` needs `imkl`, but `imkl` doesn't necessarily need MPI. We have it set up that way, just to avoid having `numpy` available after loading an MPI, which for most users is confusing, and for us is extra overhead. Following the way things are done upstream having `mpi4py` here is not a biggie, but not everyone does it that way. I think Compute Canada also does it this way. @bartoldeman ?

But I really have no horse on this race so I won't fight this any further ;-)

Reply:
yes we install mkl without interfaces and then basically used the gmkl/iimkl toolchain for numpy.
(although the present numpy installation goes via a wheel that is compiled via a script that does `module load gcc/7.3.0 imkl/imkl/2018.3.222` and can be used at runtime with pretty much any toolchain because of rpath).

JSC goes one step down and uses gcccoremkl.

Note that the missing FFTW wrappers have not really concerned us much. It's only the mpi wrappers for FFTW that are missing, and very few programs used them. I haven't found any so far on our systems which match `grep fftw_mpi` in the binaries and libraries...

Reply:
Ah spoke too soon, found one and only one match: petsc: which can use fftw mpi like explained here:
https://www.mcs.anl.gov/petsc/petsc-3.6/src/mat/examples/tests/ex158.c.html